src/pages/MarriagePage.tsx

```
@@ -1,24 +1,56 @@
 import './MarriagePage.css'
+
+import { Parallax, ParallaxLayer, IParallax } from '@react-spring/parallax'
```

ESLint

[error] 'IParallax' is defined but never used.

(@typescript-eslint/no-unused-vars)